GitUI/CommandsDialogs/BrowseDialog/GitStatusMonitor.cs

```diff
@@ -431,6 +432,7 @@ private void Update()
                         bool isSuccess = false;
                         try
                         {
+                            await Task.Delay(1).ConfigureAwait(false);
```

Comment:
Projects using vs-threading (including Git Extensions) would use this instead:

```csharp
await TaskScheduler.Default;
```

Reply:
Great
There are some further situations to investigate...